src/main/java/com/cburch/logisim/std/io/extra/Slider.java

```diff
@@ -118,46 +117,29 @@ public Object clone() {
         return null;
       }
     }
-
-    public long getCurrentValue() {
-      return this.currentvalue;
+    
+    public int getCurrentValue() {
+      final var completeValue = rightToLeft ? MAXIMUM_SLIDER_POSITION - sliderPosition : sliderPosition; 
```

Comment:
Can we have `(MAXIMUM_SLIDER_POSITION - sliderPosition)` (in brackets) for better readability?